examples/interface-to-external-code/interface_to_external_code.md

```diff
@@ -0,0 +1,7 @@
+# Interface to exteral code
+
+Ballerina supports interfaces with external code. Instead of implementing the function body, the `external`  keyword can be used. The implementation figures out how to map to the external implementation. As part of interfacing with the external implementation, Ballerina supports another basic type called `handle`. The `handle` type is basically an opaque handle that can be passed to external functions. There is no typing for `handle` and it can be added as a private member of a Ballerina class for better type safety.
```

Comment:
```suggestion
Ballerina supports interfacing to external code. This can be done by using the `external` keyword, instead of implementing the function body. The implementation figures out how to map to the external implementation. As part of interfacing to an external implementation, Ballerina supports another basic type called `handle`. The `handle` type is basically an opaque handle that can be passed to external functions. There is no typing for `handle` and it can be added as a private member of a Ballerina class for better type safety.
```